### PR TITLE
fixes cross compilation

### LINKED ISSUE #1309: [binaries] update & test cross-compilation pipeline

### PR DESCRIPTION
**Pull request by VoR0220**
Fixes #1309 by using xgo featured here: https://github.com/karalabe/xgo

The docker image is massive, but my god it just...works...and is basically a one liner to get all this done.

### CODE REVIEW

File: initialize/default_files.go [hunk 1]
```diff
@@ -273,8 +273,8 @@ We also recommend that if you are in a multi organizational chain then you would
 have some separation of the validators to be ran by the different organizations
 in the system.`
 		accountTypeDefinition.DefaultNumber = 7
-		accountTypeDefinition.DefaultTokens = 9999999999
-		accountTypeDefinition.DefaultBond = 9999999998
+		accountTypeDefinition.DefaultTokens = 99999
+		accountTypeDefinition.DefaultBond = 99999
```
Comment (zramsay) [inline initialize/default_files.go]: should be 99998 (one less)

File: initialize/default_files.go [hunk 2]
```diff
@@ -304,8 +304,8 @@ give all of the accounts needed for your experiment full accounts.
 
 If you are making a more complex chain, don't use this account type.`
 		accountTypeDefinition.DefaultNumber = 1
-		accountTypeDefinition.DefaultTokens = 99999999999999
-		accountTypeDefinition.DefaultBond = 9999999999
+		accountTypeDefinition.DefaultTokens = 9999999
```
Comment (benjaminbollen) [inline initialize/default_files.go]: why is this suddenly 9999999 ?

File: misc/release/release.sh [hunk 3]
```diff
@@ -153,15 +153,13 @@ token_check() {
...
   LDFLAGS="-X github.com/monax/cli/version.COMMIT=`git rev-parse --short HEAD 2>/dev/null`"
 
-  GOOS=linux   GOARCH=386    go build -ldflags "${LDFLAGS}" -o ${BUILD_DIR}/eris_${MONAX_VERSION}_linux_386
-  GOOS=linux   GOARCH=amd64  go build -ldflags "${LDFLAGS}" -o ${BUILD_DIR}/eris_${MONAX_VERSION}_linux_amd64
-  GOOS=darwin  GOARCH=386    go build -ldflags "${LDFLAGS}" -o ${BUILD_DIR}/eris_${MONAX_VERSION}_darwin_386
-  GOOS=darwin  GOARCH=amd64  go build -ldflags "${LDFLAGS}" -o ${BUILD_DIR}/eris_${MONAX_VERSION}_darwin_amd64
-  GOOS=windows GOARCH=386    go build -ldflags "${LDFLAGS}" -o ${BUILD_DIR}/eris_${MONAX_VERSION}_windows_386.exe
-  GOOS=windows GOARCH=amd64  go build -ldflags "${LDFLAGS}" -o ${BUILD_DIR}/eris_${MONAX_VERSION}_windows_amd64.exe
+
+  xgo -go 1.7 --targets=linux/amd64,linux/386,darwin/amd64,darwin/386,windows/amd64,windows/386 -out ${BUILD_DIR}/monax_${MONAX_VERSION} --pkg cmd/monax github.com/monax/cli
```
Comment (zramsay) [inline misc/release/release.sh]: add `-branch master`

File: definitions/account_types.go [hunk 4]
```diff
@@ -4,9 +4,9 @@ type MonaxDBAccountType struct {
 	Name          string         `mapstructure:"name" json:"name" yaml:"name" toml:"name"`
 	Description   string         `mapstructure:"description" json:"description" yaml:"description" toml:"description"`
 	TypicalUser   string         `mapstructure:"typical_user" json:"typical_user" yaml:"typical_user" toml:"typical_user"`
-	DefaultNumber int            `mapstructure:"default_number" json:"default_number" yaml:"default_number" toml:"default_number"`
-	DefaultTokens int            `mapstructure:"default_tokens" json:"default_tokens" yaml:"default_tokens" toml:"default_tokens"`
-	DefaultBond   int            `mapstructure:"default_bond" json:"default_bond" yaml:"default_bond" toml:"default_bond"`
+	DefaultNumber int64          `mapstructure:"default_number" json:"default_number" yaml:"default_number" toml:"default_number"`
+	DefaultTokens int64          `mapstructure:"default_tokens" json:"default_tokens" yaml:"default_tokens" toml:"default_tokens"`
+	DefaultBond   int64          `mapstructure:"default_bond" json:"default_bond" yaml:"default_bond" toml:"default_bond"`
```
Comment (zramsay) [inline definitions/account_types.go]: breaks a bunch of code that assumes `int`

**Pull request merged**